src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -220,6 +220,22 @@ def load_arguments(self, _):
     with self.argument_context('policy set-definition create', min_api='2017-06-01-preview', resource_type=ResourceType.MGMT_RESOURCE_POLICY) as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the new policy set definition.')
 
+    with self.argument_context('policy exemption', min_api='2020-09-01', resource_type=ResourceType.MGMT_RESOURCE_POLICY) as c:
+        c.ignore('_subscription')
```

Comment:
as you ignore subscription here, it seems there is no subscription id exposed as parameter any more?

Reply:
Yes, it is using scope parameter, which is consistent with the policy assignment command.